pkg/client/repo/chartmuseum/chartmuseumtester.go

```diff
@@ -52,7 +54,7 @@ func NewTester(t *testing.T, repo *api.Repo, emptyIndex bool, indexFile string)
 		indexFile:  indexFile,
 	}
 	s := httptest.NewServer(tester)
-	u, err := url.Parse(s.URL)
+	u, err := url.Parse(fmt.Sprintf("%s/%s", s.URL, repositroy))
```

Comment:
```suggestion
	u, err := url.Parse(fmt.Sprintf("%s/%s", s.URL, repository))
```

Reply:
Thanks for your careful code review!I have corrected my wrong spelling.